README.md

```diff
@@ -27,6 +27,76 @@ https://eui-event.vercel.app/
 - Run `yarn start dev` to start it up!
 - Go to localhost:3000 in your browser to view it!
 
+## Connect Google Sheets with your app
+
+Follow these steps to connect your google sheets file with the app.
+
+### 1. Create a google sheet
+
+#### 1. Go to https://docs.google.com/ and create a blank sheet
+#### 2. Add name, title, team, location, shortBio, pronouns, imageLink headers in the sheet.
+#### 3. Populate the sheet with some data.
```

Comment:
Can we include a sample of what the data should look like? Something like a schema, for instance: 

`name` : string, 
`imageLink`: string pointing to filepath, or url hosting image

Reply:
Noted.